docker-compose.yml

```diff
@@ -5,7 +5,10 @@
 # See comment in docker-compose.dev.yml if you want to run for development.
 
 version: '3'
-
+volumes:
+  mediawiki-image-data: {}
+  mysql-data: {}
+  mysql-backups-data: {}
```

Comment:
Would be helpful to have it general 'database-backups-data' we use already mariadb...

Reply:
Making that change could decrease comprehension of the code if we add another database in the future. This is more consistent with our ENV vars as well. @rht what are your thoughts? I know you leaned towards using `database-` as well so LMK what you think.